src/Sylius/Bundle/AdminBundle/templates/product/show/content/page_body/details/configurable_product/tab/enabled.html.twig

```diff
@@ -0,0 +1,7 @@
+{% set variant = hookable_metadata.context.variant %}
+
+<td class="text-center">
+    <span>
+        {% include '@SyliusAdmin/shared/grid/field/boolean.html.twig' with { data: variant.enabled } %}
```

Comment:
Using a grid field in a show page feels off.
We could extract the innards into a macro somewhere else, and use the macro in `field/boolean.html.twig` and the macro by itself in other places instead.